pkg/vadmin/start_db_vc.go

```diff
@@ -32,7 +32,7 @@ import (
 func (v *VClusterOps) StartDB(ctx context.Context, opts ...startdb.Option) (ctrl.Result, error) {
 	v.Log.Info("Starting vcluster StartDB")
 
-	// get the certs
+	// get the certss
```

Comment:
Why this?

Reply:
Sorry @roypaulin ,  I opened it because I just want to test my WSL after migration. I will close this PR.